x/concentrated-liquidity/lp_test.go

```diff
@@ -387,6 +387,216 @@ func (s *KeeperTestSuite) TestCreatePosition() {
 	}
 }
 
+func (s *KeeperTestSuite) TestCreatePositionNoSendUnsafe() {
+	tests := map[string]lpTest{
+		"error: non-existent pool": {
```

Comment:
I think we can also group these test cases into a list of struct called createPositionTestCases or so and de-duplicate them

Reply:
Function no longer exists so resolving this